docs/development/conventions/creating-a-release.md

```diff
@@ -18,13 +19,18 @@ $ git checkout develop
 $ git pull origin develop
 $ git checkout main
 $ git merge develop --ff
+$ git push origin main
 ```
 
 This brings the set of changes onto the `main` branch for a stable release. If that's successful, you'll be ready to publish the release.
 
 ## Publishing a release
 
-You'll use [Changesets](https://github.com/atlassian/changesets/tree/main/packages/cli) to version and publish the package. First you'll need to update the versions for all packages described in the changesets since last release:
+After pushing the commits to `main`, the [Changesets action](https://github.com/changesets/action) will create a pull request, titled `Version Packages`, with all of the package versions updated and changelogs updated. This pull request will automatically update whenever new changesets are pushed to `main`. When you're ready, you can merge the pull request and the action will publish the new version to npm for you.
+
+## Manually publishing a release
+
+You can use [Changesets](https://github.com/atlassian/changesets/tree/main/packages/cli) to version and publish the package manually. First you'll need to update the versions for all packages described in the changesets since last release:
```

Comment:
Can we clarify this sentence with the types of situations where manually publishing is desired or necessary?